### PR TITLE
Fixes #8

### LINKED ISSUE #8: filterUnchangedFiles() broken for multi-module projects
This code `String.format("%s/%s", basePath, f.getPath())` will produce wrong paths, because `basepath` points to the current module's path and `f.getPath()` is a path relative to the project execution top.

E.g for a file actually located at `/home/bochja/psb/psb/portal/src/main/java/com/acme/psp/feature/ComputerIssue.java` the changed files will contain a path like `/home/bochja/psb/psb/portal/psb/portal/src/main/java/com/acme/psp/feature/ComputerIssue.java`

### PR DESCRIPTION
**Pull request by jakub-bochenski**
Fixes #8 

Tested manually with a Git repository

### CODE REVIEW

File: src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java [hunk 1]
```diff
@@ -119,6 +119,7 @@ public void execute() throws MojoExecutionException {
   }
 
   private Set<File> filterUnchangedFiles(Set<File> originalFiles) throws MojoExecutionException {
+    MavenProject project = session.getTopLevelProject();
```
Comment (Stephan202) [inline src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java]: This variable shadows the `project` field. Maybe rename it to `topLevelProject`?

Comment (jakub-bochenski) [reply src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java]: I wanted to minimize the amount of changes

Comment (jakub-bochenski) [reply src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java]: done

File: src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java [hunk 2]
```diff
@@ -28,11 +29,7 @@
 import java.io.FileInputStream;
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (Stephan202) [inline src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java]: I guess for a minimal diff this change can be reverted then ;)

Comment (jakub-bochenski) [reply src/main/java/com/theoryinpractise/googleformatter/GoogleFormatterMojo.java]: whoops

**Comment by talios** [comment]
Having gateway timeout issues sending to oss.sonatype.org but hopefully will get 1.6.2 to central shortly.

**Comment by jakub-bochenski** [comment]
It seems the plugin is still not available in central: https://repo.maven.apache.org/maven2/com/theoryinpractise/googleformatter-maven-plugin/

**Comment by talios** [comment]
I'll try releasing again from home tonight - just tried again from our office and got timeouts again :( Having issues loading the oss.sonatype.org nexus server as well ( we're in the middle of an office move and our network is....  disrupted, assuming it's not something on Sonatype's side - I asked but haven't heard anything back yet.

**Comment by talios** [comment]
Sadly still unable to release from my home network - have commented on some tickets on https://issues.sonatype.org/ regarding someone else who was having release issues. This is frustrating :(

**Comment by talios** [comment]
Interesting - looks like releasing to oss.sonatype.org with Maven 3.3.9 worked fine! So this is on it's way to central.

**Comment by talios** [comment]
Released in 1.6.3 :)

**Pull request merged**